tools/SetupFlow/DevHome.SetupFlow/Views/PackageCatalogListView.xaml

```diff
@@ -33,11 +34,6 @@
             Spacing="27"
             Visibility="{x:Bind ViewModel.ViewAllCatalog, Mode=OneWay, Converter={StaticResource ViewAllConverter}}">
             <Grid>
-                <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="*" />
-                    <ColumnDefinition Width="auto" />

```

Comment:
If both items share the same column, wouldn't that cause an overlap on window resize (small width)?

Reply:
Ah, you're right -- no where else we use it yet resizes like that. I'll put the columns back.